### PR TITLE
Allow install even with asterisk > 17

### PR DESCRIPTION
**Pull request by Stell0**
https://github.com/nethesis/dev/issues/6124

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethserver-freepbx-conf-scl [hunk 1]
```diff
@@ -29,6 +29,7 @@ source /opt/rh/rh-php56/enable
 
 # run install
 pass=$(perl -e  'use NethServer::Password; print NethServer::Password::store('mysql');')
+
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-freepbx-conf-scl]:
```suggestion
```

File: root/etc/e-smith/events/actions/nethserver-freepbx-conf [hunk 2]
```diff
@@ -34,6 +34,17 @@ echo " *** Installing FreePBX" >> /var/log/freepbx_rpm_install.log
 
 mkdir -p /var/lib/asterisk/sounds/custom/
 
+if [[ ! -f /etc/asterisk/modules.load ]]; then
+	# write modules.load file for asterisk first start
+	echo -e '[modules]\nautoload=no' > /etc/asterisk/modules.conf
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-freepbx-conf:39]: Please include the file inside freepbx package or, better, inside asterisk one

File: root/etc/e-smith/events/actions/nethserver-freepbx-conf [hunk 3]
```diff
@@ -34,6 +34,17 @@ echo " *** Installing FreePBX" >> /var/log/freepbx_rpm_install.log
...
+
+	# Allow installation with asterisk > 17
+	sed -i 's/if (version_compare($matches\[1\], "11", "lt") || version_compare($matches\[1\], "17", "ge")) {/if (version_compare($matches[1], "11", "lt") || version_compare($matches[1], "19", "ge")) {/' /usr/src/freepbx/installlib/installcommand.class.php
+
+	# Allow retrieve conf with asterisk > 17
+	sed -i 's/if (version_compare(\$version, "11", "lt") || version_compare(\$version, "17", "ge")) {/if (version_compare(\$version, "11", "lt") || version_compare(\$version, "19", "ge")) {/' /usr/src/freepbx/amp_conf/bin/retrieve_conf
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-freepbx-conf:45]: These files are part of rpm, your are going to break package integrity.
Please create a patch to be applied by the spec

**Review by gsanchietti — CHANGES_REQUESTED** [review]

**Comment by Stell0** [comment]
patch made here https://github.com/NethServer/freepbx/pull/8 and modules.conf here https://github.com/NethServer/asterisk13/pull/29

**Pull request closed without merging**